tasks/linux-cli-http.md

```diff
@@ -56,6 +57,7 @@ When you finish this task you can proceed to the next one.
 - [HTTP: Протокол, который должен знать каждый веб-разработчик (Часть 1) (РУС)](https://code.tutsplus.com/ru/tutorials/http-the-protocol-every-web-developer-must-know-part-1--net-31177)
 - [How I taught myself to code in eight weeks](http://lifehacker.com/how-i-taught-myself-to-code-in-eight-weeks-511615189)
 - [How JavaScript works: Deep dive into WebSockets and HTTP/2 with SSE + how to pick the right path](https://blog.sessionstack.com/how-javascript-works-deep-dive-into-websockets-and-http-2-with-sse-how-to-pick-the-right-path-584e6b8e3bf7)
+- [What is HTTP?](https://www.youtube.com/watch?v=SzSXHv8RKdM) - simple video for complete beginners which will help understanding articles (2) and (3) (suggested to watch it before reading them).
```

Comment:
@ORybka 
Articles are not numbered and if the list changed (someone adds an article somewhere at the biginning of the list) reference may become broken.
If this video is recommended to be visited before those articles how about actually inserting it before referred articles? Yet description may then be shorter

`... - simple video for beginners which will help understanding other HTTP-related materials`